lib/livebook_web/live/session_live.ex

```diff
@@ -553,17 +554,37 @@ defmodule LivebookWeb.SessionLive do
       <h3 class="uppercase text-sm font-semibold text-gray-500">
         Secrets
       </h3>
+      <span class="mt-4 text-sm font-semibold text-gray-500">Available to this notebook</span>
       <div class="flex flex-col mt-4 space-y-4">
-        <%= for secret <- @data_view.secrets do %>
+        <%= for secret <- session_only_secrets(@data_view.secrets, @data_view.notebook_secrets) do %>
           <div class="flex justify-between items-center text-gray-500">
-            <span class="break-all">
+            <span class="text-sm break-all">
               <%= secret.name %>
             </span>
             <span class="rounded-full bg-gray-200 px-2 text-xs text-gray-600">
               Session
             </span>
           </div>
         <% end %>
+        <div class="w-full border-t border-gray-300 py-1"></div>
+        <div class="flex justify-between mt-4">
+          <span class="text-sm font-semibold text-gray-500">Stored in your Livebook</span>
+          <span class="text-sm font-light text-gray-500">On session</span>
+        </div>
+        <%= for secret <- @data_view.notebook_secrets do %>
+          <div class="flex justify-between items-center text-gray-500">
+            <span class="text-sm break-all">
+              <%= secret["name"] %>
+            </span>
+            <.switch_checkbox
+              name="toggle_secret"
+              checked={is_secret_on_session?(secret["name"], @data_view.secrets)}
+              phx-click="toggle_secret"
+              phx-value_secret_name={secret["name"]}
+              phx-value_secret_value={secret["value"]}
```

Comment:
```suggestion
              phx-value-secret_name={secret["name"]}
              phx-value-secret_value={secret["value"]}
```

then we should get `"secret_name"` and `"secret_value"` in the event respectively.

Reply:
I'm following this pattern everywhere and it works, except here! Even with your suggestions it's still wrong in the `handle_event`.

Reply:
Ohh, I think this may be because we are passing this into a function component and then passing the attributes through to another element.